README.md

```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
+```
 Run `make dev-prepare-database`
+```
 
 ### Run Web Server
 
-1. Run `make up`
-2. In a separate terminal, connect to the running web container, run `SERVICE=web make exec-bash`
-3. Now run: `source venv/bin/activate && ./scripts/create_user.py --username=me@mydomain.com --password=password --first_name=admin --last_name=istrator --site_admin`
+1. Run
```

Comment:
We intentionally made us if single quotes here as it result in an easier to read instruction set. Using triple quotes throws it to the next line, which doesn't read as easily.

Reply:
I agree, but I feel this way is more clear. 